ui/components/app/wallet-overview/eth-overview.js

```diff
@@ -113,6 +113,7 @@ const EthOverview = ({ className }) => {
               <Icon name={ICON_NAMES.CARD} color={IconColor.primaryInverse} />
             }
             disabled={!isBuyableChain}
+            data-testid="eth-overview-buy"
```

Comment:
Nit: we avoid using `data-testid`

Reply:
we don't anymore @jpuri -- it is *heavily* relied upon in e2e tests